axelor-account/src/main/java/com/axelor/apps/account/service/moveline/MoveLineCreateServiceImpl.java

```diff
@@ -839,6 +840,33 @@ public MoveLine createMoveLineForAutoTax(
 
     BigDecimal taxLineValue =
         taxLineBeforeReverse != null ? taxLineBeforeReverse.getValue() : taxLine.getValue();
+
+    if (percentMoveTemplate) {
+      List<MoveLine> moveLineList = move.getMoveLineList();
+      debit =
+          moveLineList.stream()
+              .filter(
+                  ml ->
+                      ml.getAccount()
+                          .getAccountType()
+                          .getTechnicalTypeSelect()
+                          .equals(AccountTypeRepository.TYPE_PAYABLE))
+              .map(it -> it.getDebit().add(it.getCredit()))
+              .reduce(BigDecimal::add)
+              .orElse(BigDecimal.ZERO);
+      credit =
+          moveLineList.stream()
+              .filter(
+                  ml ->
+                      ml.getAccount()
+                          .getAccountType()
+                          .getTechnicalTypeSelect()
+                          .equals(AccountTypeRepository.TYPE_RECEIVABLE))
+              .map(it -> it.getDebit().add(it.getCredit()))
+              .reduce(BigDecimal::add)
+              .orElse(BigDecimal.ZERO);
```

Comment:
Please create a method with an account type as parameter to compute those

Reply:
Resolved